> Could you please take a look:
> [install-x64-uwp-dbg-out.log](https://github.com/microsoft/vcpkg/files/6684382/install-x64-uwp-dbg-out.log)

I don't think it is related to libogg:

    4>D:\buildtrees\simage\src\mage-1.8.0-17f988a113.clean\src\simage_tiff.c(88,18): error C4996: 'uint16': libtiff type deprecated; please use corresponding C99 stdint.h type [D:\buildtrees\simage\x64-uwp-dbg\simage.vcxproj]

--- a/ports/libogg/vcpkg.json
+++ b/ports/libogg/vcpkg.json
@@ -1,6 +1,6 @@
 {
   "name": "libogg",
-  "version-string": "1.3.5",
+  "version": "1.3.5",
   "description": "Ogg is a multimedia container format, and the native file and stream format for the Xiph.org multimedia codecs.",
   "homepage": "https://www.xiph.org/ogg"
 }

--- a/versions/l-/libogg.json
+++ b/versions/l-/libogg.json
@@ -1,8 +1,8 @@
 {
   "versions": [
     {
-      "git-tree": "100344f3d0e17ffe5eebc2f5154b92201a93e4b4",
-      "version-string": "1.3.5",
+      "git-tree": "70cb700f89c98b87d678a3aa7a14461eff4b99e4",
+      "version": "1.3.5",
       "port-version": 0
     },
     {

